### ISSUE TITLE
Sort out aarch64 build job name format

### ISSUE THREAD

**Issue by kiwik** [theopenlab/openlab#318]
Thank you for reporting bug!

### Describe the bug
Some `aarch64` job names don't apply OpenLab [job naming notations](https://docs.openlabtesting.org/contributors/development-guide#job-naming-notations), maybe cause some confusion, we should keep consistent for these same type jobs.

### Related Job information
If the bug is related a CI job, put the job information below:
* job name:
  - containerd-build-arm64
  - envoy-build-arm64
  - envoy-build-arm64-test
  - hbase-build-arm64
  - hive-build-arm64
  - hadoop-build-arm64
  - spark-build-arm64
  - spark-aarch64-build
  - flink-build-and-test-arm64

* job build log link: NA
* error log piece: NA

### Steps to reproduce the issue
Describe how to reproduce the issue:
See everyday testing log

**Comment by Yikun** [comment]
The nodesets contains some "arm64" [1], it seems we should also change it. But it will break some online CI, so I think blew thing should to be done:
1. Keep the `arm64` and `aarch64` at first
2. Change all nodesets(in job defination) to `aarch64`.
3. Remove all arm64.

[1] https://github.com/theopenlab/openlab-zuul-jobs/blob/602ae18c7557588ed43fecc7d8c120d41989cfd9/zuul.d/nodesets.yaml#L27-L37

**Comment by ZhengZhenyu** [comment]
> The nodesets contains some "arm64" [1], it seems we should also change it. But it will break some online CI, so I think blew thing should to be done:
> 
> 1. Keep the `arm64` and `aarch64` at first
> 2. Change all nodesets(in job defination) to `aarch64`.
> 3. Remove all arm64.
> 
> [1] https://github.com/theopenlab/openlab-zuul-jobs/blob/602ae18c7557588ed43fecc7d8c120d41989cfd9/zuul.d/nodesets.yaml#L27-L37

Yes, I found this too

**Comment by Yikun** [comment]
Before nodesets.yaml is changed, we also should address nodepool config.

[cloud provider image name]
the vexxhost image should be changed to aarch64

[nodepool.yaml] 
- `labels.name`: change to `ubuntu-xenial-aarch64`
- `vexxhost-openlab.cloud-images.name`: change to `ubuntu-xenial-aarch64`
- `vexxhost-openlab.cloud-images.image-name`: change to `openlab-ubuntu-xenial-aarch64`
- `vexxhost-openlab.pools.labels.name`: `ubuntu-xenial-aarch64`
- `vexxhost-openlab.pools.lables.cloud-image`: `ubuntu-xenial-aarch64`

**Comment by Yikun** [comment]
IMO, we should do nodesets.yaml first, and then do the job defination change, otherwise we will require 2 times name format in job.name and job.nodeset . It will bring some confusing on upstream community like containerd, and envoy.

And the otherthing, I didn't see any official announcement in Linux or ARM, anyone found it? Because the project like containerd and envoy jobs are defined in their own repo, and should do a convincing proof. (The only thing I can think is that when we type the `uname -m` or `unname -a`, the linux name is the `aarch64` rather than `arm64`)

**Comment by kiwik** [comment]
@Yikun agree, I think `arm64` is more readable name for OpenLab jobs and nodeset, so I update issue content, and keep all related jobs to follow the job name format `{project}-build-arm64` and nodeset name format `{os-name}-arm64`
